ui/admin-portal/src/app/app.module.ts

```diff
@@ -28,12 +28,8 @@ import {HomeComponent} from './components/candidates/home.component';
 import {HTTP_INTERCEPTORS, HttpClientModule} from '@angular/common/http';
 import {FormsModule, ReactiveFormsModule} from '@angular/forms';
 import {ViewCandidateComponent} from './components/candidates/view/view-candidate.component';
-import {
```

Comment:
Arrgggh! This is just reformatting. Can you create an issue to try and standardize on our Intellij formatting to avoid this continuous reformatting.